omero/developers/JavaGateway.txt

```diff
@@ -0,0 +1,19 @@
+Java Gateway documentation
```

Comment:
You don't need 'documentation' in the title, it's rather self explanatory given this is in the docs